go/tao/cached_guard_test.go

```diff
@@ -177,3 +178,125 @@ func TestCachingDatalogReload(t *testing.T) {
 	}
 	<-ch
 }
+
+// Test that a client can correctly verify that the server is allowed to
+// execute according to the policy. The policy is set up and the policy
+// key is used to attest to the identity of the server. The attestation
+// includes an endorsement of the service itself. The client verifies the
+// endorsement and adds the predicate to the policy before it checking.
+func TestCachingDatalogValidatePeerAttestation(t *testing.T) {
+	var network, addr string
+	var ttl int64
```

Comment:
instead of defining ttl in two lines, do

ttl := int64(1)

similarly for network/addr strings